frontend/src/container/GridGraphLayout/Graph/FullView/contants.ts

```diff
@@ -0,0 +1,16 @@
+export enum ColumnsTitle {
```

Comment:
there should be a mapping between key and title where key is unique

Reply:
Okay

Reply:
Okay got it now, changing it.